api/resources_portal/management/commands/populate_test_database.py

```diff
@@ -27,6 +27,8 @@ def add_class_to_database(class_json, Class):
                     element["updated_at"] = dateparse.parse_datetime(element["updated_at"])
                 if "created_at" in element:
                     element["created_at"] = dateparse.parse_datetime(element["created_at"])
+                if "date_joined" in element:
+                    element["date_joined"] = dateparse.parse_datetime(element["date_joined"])
 
```

Comment:
```suggestion
                if "embargo_date" in element:
                    element["embargo_date"] = dateparse.parse_datetime(element["embargo_date"])
```

Reply:
This isn't necessary, see: https://github.com/AlexsLemonade/resources-portal/pull/151#issuecomment-622525988